tests/ensure_provider_tests.py

```diff
@@ -91,6 +91,12 @@
 ansible_distribution == 'CentOS' or\n     ansible_distribution_major_version > '32' \
 and ansible_distribution == 'Fedora'",
     },
+    "playbooks/tests_wireless_wpa3_sae.yml": {
+        "comment": "# SAE has not been supported by NetworkManager 1.18.8 on \
+RHEL 7(dist-tag). Failed in setting up mock wifi on RHEL 8",
```

Comment:
What do you mean with `(dist-tag)` - it does not seem to make sense to me.

Reply:
Removed.